Code/Mantid/Framework/DataHandling/src/ProcessDasNexusLog.cpp

```diff
@@ -445,7 +447,9 @@ void ProcessDasNexusLog::writeLogtoFile(API::MatrixWorkspace_sptr ws,
   Kernel::Property *log = ws->run().getProperty(logname);
   Kernel::TimeSeriesProperty<double> *tslog =
       dynamic_cast<Kernel::TimeSeriesProperty<double> *>(log);
-  std::vector<Kernel::DateAndTime> times = tslog->timesAsVector();
+  std::vector<Kernel::DateAndTime> times;
```

Comment:
Yes, this is better, and raising a exception looks like the proper thing to do. After all tslog is cast from a log entry given by the user, so it should be reported if it's not a valid one.